### PR TITLE
Add file parsing apache logs for download stats.

### PR DESCRIPTION
**Pull request by AlexandraDobrescu**

### CODE REVIEW

File: rstats/SQLitePlatformDistribution.C [hunk 1]
```diff
@@ -0,0 +1,57 @@
+ #include <TSQLiteServer.h>
+ #include <TSQLiteResult.h>
+ #include <TSQLiteStatement.h>
+ #include <TSQLColumnInfo.h>
+ #include <TList.h>
```
Comment (Axel-Naumann) [inline rstats/SQLitePlatformDistribution.C]: We don't need this include I believe?

File: rstats/SQLitePlatformDistribution.C [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ #include <TSQLTableInfo.h>
+ #include <TSQLRow.h>
+ #include <TAttAxis.h>
```
Comment (Axel-Naumann) [inline rstats/SQLitePlatformDistribution.C]: Needed?

File: rstats/SQLitePlatformDistribution.C [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ #include <TString.h>
+
+void SQLitePlatformDistribution(){
+
+TSQLServer *db = TSQLServer::Connect("sqlite:///home/adobresc/Documents/root_download_stats.sqlite", "", "");
```
Comment (Axel-Naumann) [inline rstats/SQLitePlatformDistribution.C]: Indent?

File: rstats/SQLitePlatformDistribution.C [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+   const char *rootPlatform = "SELECT Platform FROM accesslog;";
+
+  TSQLResult *rootPlatformRes = db->Query(rootPlatform);
+
+  TH1F *hrootPlatform = new TH1F("hrootPlatform", "Platform Distribution", 7, 0, -1);
+  TH1F *shorthrootPlatform = new TH1F("shorthrootPlatform", "Short Platform Distribution", 7, 0, -1);
+
+  while (TSQLRow *row = rootPlatformRes->Next()) {
+      TString rowPlatform(row->GetField(0));
```
Comment (Axel-Naumann) [inline rstats/SQLitePlatformDistribution.C]: For C++ we use an indentation of three spaces.

File: rstats/SQLitePlatformDistribution.C [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      TString Platform(rowPlatform);
```
Comment (Axel-Naumann) [inline rstats/SQLitePlatformDistribution.C]: Can we move them down, i.e. get rid of them here and
```
if ( rowPlatform.Contains("win32") ){
   shorthrootPlatform->Fill(rowPlatform(0,5), 1);
```

Comment (Axel-Naumann) [inline rstats/SQLitePlatformDistribution.C]: `Platform` should be lower-case.

File: rstats/SQLitePlatformDistribution.C [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      } else if ( rowPlatform.Contains("IRIX64") ){
+        shorthrootPlatform->Fill(Platform_1,1);
+      }
+
+      hrootPlatform->Fill(Platform,1);
+           
+      delete row;
+   }
+
+  TCanvas *PlatformDistributionHistogram = new TCanvas();
```
Comment (Axel-Naumann) [inline rstats/SQLitePlatformDistribution.C]: `PlatformDistributionHistogram` should be lower-case.

File: rstats/SQLiteTimeVersionOfRoot.C [hunk 7]
```diff
@@ -0,0 +1,48 @@
+ #include <TSQLiteServer.h>
+ #include <TSQLiteResult.h>
+ #include <TSQLiteStatement.h>
+ #include <TSQLColumnInfo.h>
+ #include <TList.h>
```
Comment (Axel-Naumann) [inline rstats/SQLiteTimeVersionOfRoot.C]: Check includes, please!

**Comment by ananelle** [comment]
delete logs

**Pull request merged**